splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/InListMultiprobeIT.java

```diff
@@ -32,8 +39,19 @@
 /**
  * Test the IN list predicates with the multiprobe index scan
  */
+@RunWith(Parameterized.class)
+@Category(SerialTest.class)
```

Comment:
Because the test involves calling syscs_set_global_database_property.  If the tests are run in parallel, the call might happen at the wrong time for one of the tests and change the results, which then might not match expected results.

Reply:
I see, I overlooked that one test case has the database property change.